src/v0/destinations/google_adwords_offline_conversions/utils.js

```diff
@@ -272,7 +272,9 @@ const getAddConversionPayload = (message, Config, eventLevelConsentsData) => {
         ? sha256(email.trim()).toString()
         : email,
     phone:
-      hashUserIdentifier && isDefinedAndNotNull(phone) ? sha256(phone.trim()).toString() : phone,
+      hashUserIdentifier && isString(phone) && isDefinedAndNotNull(phone)
```

Comment:
```suggestion
      hashUserIdentifier && isDefinedAndNotNull(phone) && isString(phone)
```
We should check the existence of the phone, then check if that is a string or not.